content/developer/tutorials/server_framework_101/02_newapp.rst

```diff
@@ -1,99 +1,279 @@
+not yet been applied to the database. To do so, you must add `-u real_estate` to the server start-up
+command and restart the server. The :option:`-u <odoo-bin --update>` argument tells the server to
+upgrade the specified modules at start-up.
+
+Inspect the SQL table
+=====================
+
+Earlier, we quickly introduced models as a convenient way to store and handle data in Odoo. These
+models not only define the structure and behavior of data in Python, but they also correspond to SQL
+tables in the database. The `_name` attribute of their model is taken (with dots replaced by
+underscores) as the name of the corresponding tables.
+
+The same is true for fields that become columns in the SQL table of their model. The name of the
+class attribute representing the field is taken as column name while the field's class determines
+the column type.
+
+Once the server is running again, let's take a look in the database and see how the model and fields
+you created translate into a new SQL table. We will use `psql`, the CLI allowing to browse and
+interact with PostgreSQL databases.
+
+.. exercise::
+
+   #. In your terminal, execute the command :command:`psql -d tutorials`.
+   #. Enter the command :command:`\\d real_estate_property` to print the description of the
+      `real_estate_property` table.
+   #. For each field of the `real.estate.property` model, understand how the field's attributes alter
+      the table.
+   #. Enter the command :command:`exit` to exit `psql`.
+
+.. spoiler:: Solution
+
+   .. code-block:: text
+
+      $ psql -d tutorials
+
+      tutorials=> \d real_estate_property
+                                                  Table "public.real_estate_property"
+            Column       |            Type             | Collation | Nullable |                     Default
+      -------------------+-----------------------------+-----------+----------+--------------------------------------------------
+       id                | integer                     |           | not null | nextval('real_estate_property_id_seq'::regclass)
+       floor_area        | integer                     |           |          |
+       create_uid        | integer                     |           |          |
+       write_uid         | integer                     |           |          |
+       name              | character varying           |           | not null |
+       type              | character varying           |           | not null |
+       availability_date | date                        |           |          |
+       description       | text                        |           |          |
+       has_garden        | boolean                     |           |          |
+       has_garage        | boolean                     |           |          |
+       create_date       | timestamp without time zone |           |          |
+       write_date        | timestamp without time zone |           |          |
```

Comment:
is it worth to mention that there are additional columns that are added in the table for us like `id`. `create_date` and so on?

Reply:
Yes! Will do right after :)